kube-apiserver/kube-apiserver-service.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   - port: 6443
     protocol: TCP
     targetPort: 6443
-    nodePort: 31123 # this is random but fixed so we can create a static LB (e.g. haproxy) to point to it
+    nodePort: ${API_NODEPORT}
```

Comment:
this should be `EXTERNAL_API_PORT`.  one of the places i forgot to template.

Reply:
Actually I meant to have API_NODEPORT in the sample config and make that the value we set here. EXTERNAL_API_PORT in the AWS setup will always be 6443 (since it's what we use to generate the kubeconfig)